osu.Game/Graphics/Cursor/CursorTrail.cs

```diff
@@ -116,7 +116,7 @@ protected override bool OnMouseMove(InputState state)
             float distance = diff.Length;
             Vector2 direction = diff / distance;
 
-            float interval = size.X / 2;
+            float interval = (size.X / 2) * 0.9f;

```

Comment:
* 0.45 without the division by 2. Don't want people to do arithmetic in their head when reading the code.